src/client/api/event.c

```diff
@@ -27,31 +28,6 @@ static __thread bool		ev_thpriv_is_init;
  */
 static uint32_t ev_prog_timeout;
 
-#define EQ_WITH_CRT
-
-#if !defined(EQ_WITH_CRT)
```

Comment:
This is probably the whole point, but this was never used, right? Because we define EQ_WITH_CRT always